test/playwright/e2e/load-more.spec.ts

```diff
@@ -0,0 +1,130 @@
+import { expect, Page, test } from '@playwright/test'
+
+import {
+  goToSearchTerm,
+  renderModes,
+} from '~~/test/playwright/utils/navigation'
+import { mockProviderApis } from '~~/test/playwright/utils/route'
+
+import { AUDIO, IMAGE, SupportedMediaType } from '~/constants/media'
+
+test.describe.configure({ mode: 'parallel' })
+
+test.beforeEach(async ({ context }) => {
+  await mockProviderApis(context)
+})
+
+const loadMoreButton = 'button:has-text("Load more")'
+
+const openSingleMediaView = async (
+  page: Page,
+  mediaType: SupportedMediaType
+) => {
+  const contentLinkSelector =
+    mediaType === IMAGE ? 'See all images' : 'See all audio'
+  return await Promise.all([
+    page.waitForNavigation(),
+    page.click(`text=${contentLinkSelector}`),
+  ])
+}
+/**
+ * Cases, check both SSR and CSR:
+ * 1. All content view with more than 1 page of results for each media type:
+ *  - should have a Load more button.
+ *  - each individual media type view should have a Load more button.
+ *  - when button is clicked, should fetch all media types.
+ * 2. All content view with results for images, but only 1 page of results for audio:
+ *  - should have a Load more button.
+ *  - when button is clicked, should fetch only images.
+ *  - image view should have a Load more button, but audio view should not.
+ * 3. All content view with results for images, but no results for audio:
+ *  - should have a Load more button.
+ *  - when button is clicked, should fetch only images.
+ *  - image view should have a Load more button.
+ */
```

Comment:
Although it is unlikely to differ in behaviour, but for completeness' sake, we should add the opposite of cases 2 and 3 where audio results are more than image.

Reply:
I agree, but I couldn't find an example search term for that 😆 I guess I should mock the response.